lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -215,4 +216,10 @@ defmodule ExUnit.CLIFormatter do
       _ -> 80
     end
   end
+
+  defp print_logs(""), do: nil
+
+  defp print_logs(output) do
+    IO.puts(["The following output was logged:\n" | output])
```

Comment:
The indentation might be tricky, e.g:
```
     stacktrace:
       test/ex_unit_test.exs:140

The following output was logged:

21:28:37.453 [debug] three
```
We can indent header easily, but the `output` here is just binary, I think we can't safely rewrite it (as it can contain everything). Probably, __1)__ we can provide `:format` option for capture logger with needed indentation (in that case we should consider user's `:format` option); __2)__ as another option - wrap output into something like this:
```
     stacktrace:
       test/ex_unit_test.exs:140

     The following output was logged:
===
21:28:37.453 [debug] three
21:28:37.453 [debug] four
===
```
What are your thoughts on it?